web/src/timerender.ts

```diff
@@ -18,7 +18,8 @@ import * as util from "./util.ts";
 
 let next_timerender_id = 0;
 
-export let display_time_zone = new Intl.DateTimeFormat().resolvedOptions().timeZone;
+export const browser_time_zone = new Intl.DateTimeFormat().resolvedOptions().timeZone;
```

Comment:
I'm not sure it's correct to define this as `const`; we might want to have `is_browser_timezone_same_as` recompute it later. I don't know exactly how this works, but I think if you change your timezone in your browser's settings, it doesn't reload existing tabs, but this will return a new value?

Reply:
Changing the timezone indeed doesn't reload tabs, and changes in the timezone are dynamically reflected. 

We can make `browser_time_zone` a function that always calculates the current timezone.